src/Sarif.Multitool/Sarif.Multitool.csproj

```diff
@@ -30,7 +30,10 @@
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net472</TargetFrameworks>
+    <!-- 
+      net462 is the current minimal supported .NET framework.
+    -->
```

Comment:
Is there a reason this comment is split to 3 lines? The others in the file appear to be single line comments.

Reply:
The reason is 'I am only human' and prone to inconsistencies. :) Fixed, thanks for the note.